_data/opencollective.json

```diff
@@ -2392,7 +2413,7 @@
     "last_payment": 0.0,
     "all_time": 640.0,
     "since": "Dec 28, 2018",
-    "time_last_payment": "Jun 23, 2024",
+    "time_last_payment": "Jul 3, 2024",
```

Comment:
This looks odd: `time_last_payment` is updated, but `last_payment` is 0. That doesn't make much sense.

Reply:
that's what OC data says. it happens all the time 🤷